src/profile/scripts/directives.js

```diff
@@ -24,7 +24,9 @@
                             scope.fields = response.data.actions.PUT;
                         });
 
-                    scope.editProfile = function () {
+                    scope.editProfile = function (localStorageVar) {
```

Comment:
Using `localStorage` in a name of an object that's not accessing `Window.localStorage` can be confusing. Perhaps something like `scopeObject`. That way it'll give a clue to where the argument is used, and the type. Naming things is hard!